tests/integration/components/conversations/conversation-part-closed-test.js

```diff
@@ -1,10 +1,31 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import component from 'code-corps-ember/tests/pages/components/conversations/conversation-part-closed';
+import stubService from 'code-corps-ember/tests/helpers/stub-service';
+
+let page = PageObject.create(component);
+
+function renderPage() {
+  page.render(hbs`
+    {{conversations/conversation-part-closed
+      author=author
+      closedAt=closedAt
+    }}
+  `);
 
 moduleForComponent('conversations/conversation-part-closed', 'Integration | Component | conversations/conversation part closed', {
   integration: true
 });
 
```

Comment:
Would love feedback on test statements on 21 + 25.

Reply:
For future reference, you should comment below those lines so we can see them in this view. Also, would be helpful to have some direction in terms of what you want feedback on specifically?